app/src/test/java/com/android/unio/model/firestore/HydrationAndSerializationTest.kt

```diff
@@ -108,7 +115,8 @@ class HydrationAndSerializationTest {
     assertEquals(association.name, serialized["name"])
     assertEquals(association.fullName, serialized["fullName"])
     assertEquals(association.description, serialized["description"])
-    assertEquals(association.members.uids, serialized["members"])
+    assertEquals(mapUsersToRoles(association.members), serialized["members"])
+    assertEquals(mapRolesToPermission(association.roles), serialized["roles"])
     assertEquals(association.image, serialized["image"])
     assertEquals(association.events.uids, serialized["events"])
```

Comment:
These strings could be changed to Association::parameter.name, but as the file was already entirely written with hardcoded keys, it can also be the object of a later very small refactor

Reply:
I will do a big refactor for all the tests